src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Media/ImageBrushTests/ImageBrushLocalAsset.xaml

```diff
@@ -24,10 +24,10 @@
 						BorderBrush="Sienna"
 						BorderThickness="3">
 					<Border.Background>
-						<ImageBrush ImageSource="/Assets/Diagnostics/avatar.png"/>
```

Comment:
Why change this, and not add a new one ?

Reply:
Because this asset doesn't exists.